react/hooks/useMobileAppResume.js

```diff
@@ -0,0 +1,18 @@
+import { useEffect } from 'react'
+import { isMobileApp } from 'cozy-device-helper'
+
+const useMobileAppResume = cb => {
+  useEffect(() => {
+    if (isMobileApp()) {
+      document.addEventListener('resume', cb, false)
+    }
+
+    return () => {
+      if (isMobileApp()) {
+        document.removeEventListener('resume', cb, false)
+      }
+    }
+  }, [cb])
+}
```

Comment:
Nit: I am not sure it is necessary to check whether it is a mobile app before subscribing to the event. AMHA subscribing to the resume event is fairly cheap and does not consume a lot of resource.

Nit: A useEventListener hook could be used here to have more flexibility in the future.

Reply:
I changed the useMobileAppResume hook to useEventListener. This addresses both nits: we don't check the platform, and we have the flexibility.